tests/cloud_playbooks/roles/packet-ci/templates/inventory.j2

```diff
@@ -4,6 +4,9 @@ instance-{{ loop.index }} ansible_ssh_host={{instance.stdout}}
 {% endfor %}
 
 {% if mode is defined and mode in ["separate", "separate-scale"] %}
+[kube_control_plane]
+instance-1
+
 [kube-master]
 instance-1
```

Comment:
why do you keep `[kube-master]` here ? (multiple occurrences in this file)

Reply:
Yes, we need to keep this for passing upgrading test which is from stable version.

Reply:
Maybe put a `#TODO: remove after 2.1X`